src/Phinx/Wrapper/TextWrapper.php

```diff
@@ -107,6 +110,45 @@ public function getRollback($env = null, $target = null)
         return $this->executeRun($command);
     }
 
+    /**
+     * Set the environment
+     *
+     * @param string $environment
+     */
+    public function setEnvironment($environment = null) {
```

Comment:
This was so that we have an alternative option in setting the environment, based on user preference. They could either write it as this:
```php
<?php
$app = new PhinxApplication();
$wrap = new TextWrapper($app, array(
    ...
));
```

Or it could be done by:

```php
<?php
$app = new PhinxApplication();
$wrap = new TextWrapper($app);
$wrap->setEnvironment('Environment Name');
```

They would both do the same thing, but its just a different way for the user to write it.